scripts/doQC7.sh

```diff
@@ -6,15 +6,20 @@ helpstring="Usage: doQC7.sh -c [DetName]
         -c chamber name, without '/' characters, e.g. 'GE11-X-S-CERN-0007'
         -d Debugging flag, prints additional information
         -f starting step to begin testConnectivity.py with, defaults to 1, options [1,5]
+        -g GEM detector type (i.e. 'long' or 'short')
```

Comment:
I think we better use another letter for this option, e.g. `-t` as `-g` is used for optical links elsewhere and may cause a confusion

Reply:
The `-g` option is legacy and in almost all cases I've slowly replaced `-g` with a positional argument called `link`.

Also users reading the help menu would work here! :D 

But if you feel strongly I can change it.